sema/discovery/discovery.py

```diff
@@ -155,6 +158,11 @@ def _add_triples_from_text(self, content, mimetype, source_url):
     def _extract_triples_from_response(self, resp: Response):
         # note we can be sure the response is ok, as we checked that before
         ctype_header = resp.headers.get("Content-Type", None)
+
+        if not ctype_header:
```

Comment:
By returning early when the Content-Type header is missing, the code avoids executing further logic that depends on this header. This prevents potential errors or incorrect behavior.
This is because without this logic the line 166 could produce an error